pages/learn.vue

```diff
@@ -82,11 +87,22 @@
             </client-only>
           </fieldset>
           <section class="the-learning-resources-list__results">
-            <TheCarefulExplanationForBeginner v-if="isAMinuteForBeginner" />
-            <TheCarefulExplanationForAdvanced v-if="isAMinuteForAdvanced" />
+            <TheCarefulExplanationForBeginner
+              v-if="isShowingOneMinuteFor(learnLevels.beginner) && !isShowingEverything"
+              class="the-learning-resources-list__item"
+              :compact="isShowingMoreResources"
+              url="/learn/?learnLevel=beginner&amp;timeScale=1%20minute"
+            />
+            <TheCarefulExplanationForExperts
```

Comment:
Just for curiosity: why did you change "Advanced" for "Expert" in some parts of the code?

Reply:
Because of the title of the text, to make it match with:
"A Carefully Worded Summary of Quantum Computing for Experts"